src/DotNetStatus/Startup.cs

```diff
@@ -19,9 +18,12 @@ public class Startup
 
         public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
         {
+            string configName = env.IsDevelopment() ? "config.debug.json" : "config.json";
```

Comment:
The convention for this is to load the config.json, and then load the config.debug.json to overwrite any values depending on if you want it for development.

The current ASP.NET 5 templates in RC1, use the following load pattern:
```csharp
// Set up configuration sources.
var builder = new ConfigurationBuilder()
	.AddJsonFile("appsettings.json");

if (env.IsEnvironment("Development"))
{
	// This will push telemetry data through Application Insights pipeline faster, allowing you to view results immediately.
	builder.AddApplicationInsightsSettings(developerMode: true);
}

builder.AddEnvironmentVariables();
_configuration = builder.Build().ReloadOnChanged("appsettings.json");
```